examples/lib/stories/components/composability.dart

```diff
@@ -29,32 +29,26 @@ class ParentSquare extends Square with HasGameRef {
       Square(Vector2(70, 200), Vector2(50, 50), angle: 5),
     ];
 
-    children.forEach((c) => addChild(c, gameRef: gameRef));
+    addAll(children);
   }
 }
 
-class Composability extends BaseGame with TapDetector {
-  late ParentSquare _parent;
+class Composability extends BaseGame with HasDraggableComponents {
```

Comment:
Still fiddling around with how I want to have the `GameInGame` example, since that moves a `Draggable` component from one game to this one. But I will probably remove the mixin in the end anyways since it will be confusing with the spinning.